milestone2.ts

```diff
@@ -150,11 +171,11 @@ let updateVolunteer: UpdateVolunteer; // Code here
 
 type GetVolunteerInfo = (vol: Volunteer) => String;
 let getVolunteerInfo: GetVolunteerInfo = (vol) => {
-  let name, age, city; // Code here
-  return `${name} is ${age} years old and lives in ${city}`;
+  let [name, age, city] = [vol.name, vol.age, vol.city]; // collects name age and city of object
```

Comment:
Close, but you used array destructuring and we wanted [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring)